scripts/azure-windows-release.yml

```diff
@@ -12,11 +12,16 @@ steps:
     mkdir $env:AGENT_BUILDDIRECTORY\build
     cd $env:AGENT_BUILDDIRECTORY\build
 
-    if ($env:imageName -eq "vs2017-win2016") {
-      $env:Path += ";C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
+    if ($env:imageName -eq "windows-2019") {
+      $VSCategory = "Enterprise" # alternate 'Community'
+      if (!(Test-Path "C:\Program Files (x86)\Microsoft Visual Studio\2019\${VSCategory}\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin")) {
```

Comment:
If paths continue to be a problem, there's a number of scripts called `vsvars.bat`, with variations, that set up path variables for Visual Studio. I don't know if one of them always sets up a directory for the CMake executable, but I imagine there's something that does, now that VS has a certain amount of CMake support in it.